.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,14 @@
 ---
-Link to JIRA issue: https://issues.apache.org/jira/browse/AIRFLOW-XXXX
+Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)
 
 - [ ] Description above provides context of the change
-- [ ] Commit message starts with `[AIRFLOW-NNNN]`, where AIRFLOW-NNNN = JIRA ID*
+- [ ] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
```

Comment:
```suggestion
- [ ] Commit message/PR title starts with `[AIRFLOW-NNNN]`. NNNN = JIRA ID<sup>*</sup>
```
Sorry I am mathematician 🤷🏼‍♂️

Reply:
But AIRFLOW-NNNN IS the Jira ID :).

Reply:
You can also have INFRA-NNNN and others. So mathematically speaking "AIRFLOW-NNNN" is the ID 😄